lib/PlaybackArea.tsx

```diff
@@ -14,16 +15,24 @@ export type PlayerNativeElement =
 
 const debugLog = debug('msp:api')
 
-export const AXIS_IMAGE_CGI = 'jpg'
-export const AXIS_VIDEO_CGI = 'mjpg'
-export const AXIS_MEDIA_AMP = 'media'
+export enum AxisApi {
+  'AXIS_IMAGE_CGI' = 'AXIS_IMAGE_CGI',
+  'AXIS_MEDIA_AMP' = 'AXIS_MEDIA_AMP',
+  'AXIS_MEDIA_CGI' = 'AXIS_MEDIA_CGI',
+}
 
-export type Format = 'H264' | 'MJPEG' | 'JPEG'
+export enum Format {
+  'RTP_H264' = 'RTP_H264',
+  'RTP_JPEG' = 'RTP_JPEG',
+  'JPEG' = 'JPEG',
+  'MP4_H264' = 'MP4_H264',
+}
 
 export const FORMAT_API = {
```

Comment:
type as `Record<Format, AxisApi>`?

Reply:
there are a few other places like this too

Reply:
Which other places?

Reply:
Added one more comment for the other place I found